src/style.css

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+}
+
+#header {
+  background-color: blue;
+}
+
+#header:hover {
+  height: 200px;
+}
+
+.important {
+  background-color: red;
+}
+
+.important:hover {
+  height: 200px;
+}
+
+.interesting {
+  background-color: green;
+}
+
+.interesting:hover {
+  height: 200px;
+}
+
+.surprising {
+  background-color: orange;
+}
+
+.surprising:hover {
+  height: 200px;
```

Comment:
i think you dont need repeat it for every class

Reply:
done